aws/resource_aws_rds_cluster.go

```diff
@@ -432,6 +471,97 @@ func resourceAwsRDSClusterCreate(d *schema.ResourceData, meta interface{}) error
 
 		log.Printf("[DEBUG]: RDS Cluster create response: %s", resp)
 
+	} else if v, ok := d.GetOk("s3_import"); ok {
+		if _, ok := d.GetOk("master_password"); !ok {
+			return fmt.Errorf(`provider.aws: aws_db_instance: %s: "password": required field is not set`, d.Get("name").(string))
```

Comment:
`"password"` should be `"master_password"` 😄  Same with username one below

Reply:
We use a special wrapper to parallelize the acceptance testing in our CI environment: https://github.com/jen20/teamcity-go-test/

It'd probably be nice if we offered something similar for local testing.